README.md

```diff
@@ -8,7 +8,7 @@ During this project, we will build a graphical user interface for controlling a
 
 The GUI will incorporate a videofeed, ability to connect to a ROV simulation and use a subset of the [IMC protocol](https://www.lsts.pt/toolchain/imc) for communication.
 
-###### This project is a continuation of a customer-driven project from last year, which can be found [here](https://github.com/Kpro11/Aqeous), however rewritten with web technologies.
+###### This project is a continuation of a customer-driven project from last year, which can be found [here](https://github.com/Kpro11/Aqueous), however rewritten with web technologies.
```

Comment:
This is the repo from last year, so this should not be changed

Reply:
Good catch!